app/src/main/AndroidManifest.xml

```diff
@@ -2,16 +2,21 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="comcmput301f16t01.github.carrier">
 
-    <!-- FOUR BELOW ARE USED FOR ELASTIC SEARCH -->
+    <!-- USED FOR ELASTIC SEARCH -->
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_PHONE_STATE" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.INTERNET" />
+
+    <!-- USED FOR LOCATION -->
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
+    <!-- USED TO CALL-->
```

Comment:
Maybe say "USED TO MAKE PHONE CALLS". For a second I was thinking function calls :P